commitlint.config.js

```diff
@@ -2,4 +2,7 @@ module.exports = {
   extends: [
     '@commitlint/config-angular',
   ],
+  rules: {
+    'body-max-line-length': [ 0 ],
```

Comment:
Can this just not be `'body-max-line-length': 0`?

Reply:
nope. I tried running it like that and it throws errors
```
husky > commit-msg (node v14.15.0)
/Users/saihajpreetsingh/Desktop/shabados/docs/node_modules/@commitlint/cli/lib/cli.js:100
        throw err;
        ^

Error: config for rule body-max-line-length must be array, received 0 of type number
    at Object.lint [as default] (/Users/saihajpreetsingh/Desktop/shabados/docs/node_modules/@commitlint/lint/lib/lint.js:84:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Promise.all (index 0)
    at async main (/Users/saihajpreetsingh/Desktop/shabados/docs/node_modules/@commitlint/cli/lib/cli.js:151:21)
```